src/App.tsx

```diff
@@ -1,9 +1,27 @@
 import './App.scss';
+import {
+  HashRouter as Router, Navigate, NavLink, Route, Routes,
+} from 'react-router-dom';
+import { PeoplePage } from './components/PeoplePage';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+export const App = () => {
```

Comment:
You already have default `export` at the bottom
```suggestion
const App = () => {
```